test/node-unit/mocha.spec.js

```diff
@@ -1,38 +1,39 @@
 'use strict';
 
-const path = require('path');
 const Mocha = require('../../lib/mocha');
 const utils = require('../../lib/utils');
 
 describe('Mocha', function() {
   const opts = {reporter: utils.noop}; // no output
-  const testFiles = [
-    __filename,
-    path.join(__dirname, 'cli', 'config.spec.js'),
-    path.join(__dirname, 'cli', 'run.spec.js')
-  ];
-  const resolvedTestFiles = testFiles.map(require.resolve);
 
   describe('#addFile', function() {
     it('should add the given file to the files array', function() {
       const mocha = new Mocha(opts);
-      mocha.addFile(__filename);
-      expect(mocha.files, 'to have length', 1).and('to contain', __filename);
+      mocha.addFile('some-file.js');
+      expect(mocha.files, 'to have length', 1).and(
+        'to contain',
+        'some-file.js'
+      );
     });
 
     it('should be chainable', function() {
       const mocha = new Mocha(opts);
-      expect(mocha.addFile(__filename), 'to be', mocha);
+      expect(mocha.addFile('some-file.js'), 'to be', mocha);
     });
   });
 
   describe('#loadFiles', function() {
     it('should load all files from the files array', function() {
+      this.timeout(1000);
       const mocha = new Mocha(opts);
 
-      testFiles.forEach(mocha.addFile, mocha);
+      mocha.addFile(require.resolve('../../package.json'));
```

Comment:
Hasn't _package.json_ been loaded by the main thread already?

Reply:
yeah, good point.  I've rewritten some of these tests in 798155b3. it's not a great idea to rely on test fixtures like this anyway, so I created a couple.  these still aren't _unit_ tests--because of the reliance on `require()` and I/O--but they are closer now.